### PR TITLE
aws and google cloud analytics and notification api 

### PR DESCRIPTION
**Pull request by PratikDhanave**
aws and google cloud analytics and notification api 
docs 
examples

### CODE REVIEW

File: database/bigtable/bigtable.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+PACKAGE DOCUMENTATION
```
Comment (OddCN) [inline database/bigtable/bigtable.md]: Does it this file (database/bigtable/bigtable.md) repeat with APIDocs/bigtable.md ?

**Comment by PratikDhanave** [comment]
updated thank @OddCN for review :+1:

**Comment by PratikDhanave** [comment]
@OddCN updated thanks  for review +1

File: analytics/redshift/utils.go [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+			n := strconv.Itoa(i)
+
+			prefix := "VpcSecurityGroupIds.VpcSecurityGroupId." + n
+
+			createClusterpram[prefix] = createCluster.vpcSecurityGroupIds[i]
+		}
+	}
+
+	if len(createCluster.tagKeys) != 0 {
+
```
[the remark this answers is not in the document]
Comment (PratikDhanave) [reply analytics/redshift/utils.go]: @rehrumesh okay

**Pull request merged**